src/environments/environment.ts

```diff
@@ -3,9 +3,20 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  firebase: {
+    apiKey: 'AIzaSyBOVaCDXt3F8jLICR1vZc83uRbwoE0pSHs',
```

Comment:
Nazarii, please delete API Key from git.

Reply:
removed in new commit